crm_rma_prodlot_supplier/models/stock_production_lot.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Vauxoo
+#    Author: Yanina Aular, Osval Reyes
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import api, fields, models
+
+
+class StockProductionLot(models.Model):
+
+    _inherit = 'stock.production.lot'
+
+    supplier_id = fields.Many2one('res.partner', string='Supplier',
+                                  help="Supplier of good in claim")
+
+    supplier_invoice_line_id = \
+        fields.Many2one('account.invoice.line',
+                        string='Supplier Invoice Line',
+                        help="Supplier invoice with the "
+                             "purchase of goods sold to "
+                             "customer")
+
+    @api.model
+    def default_get(self, fields_list):
+        """
+        Set partner when product lot is created
+        @param fields_list: record values
+        @return: return record
+        """
+        res = super(StockProductionLot, self).default_get(fields_list)
```

Comment:
@pedrobaeza The heuristic complicated is for to search the supplier invoice line that it belongs to the product with serial number that the user wanted transferring. you can see, if picking is incoming type then the product received is new ware, besides, for that product will be created a new serial number. Across purchase line of stock move is searched the supplier invoice line for the serial number. I do not know if I explain well. 
It's possible that this code can be improved.... @osvalr do you help me? please. When you're improving this code, let me know.

FYI @nhomar